### PR TITLE
Improve Netlify CMS

### PR DESCRIPTION
**Pull request by gabrielpoca**
* Adds preview using the blog post's page components.
* Adds editorial workflow.

I had to update the way the post's path worked to make the editorial workflow work. It's a long story, but we can't have "/" in the post's path.

### CODE REVIEW

File: config/gatsby/plugins/rssFeed.js [hunk 1]
```diff
@@ -43,9 +43,9 @@ module.exports = () => [
                 ],
                 date,
                 description: intro,
-                guid: slug,
+                guid: url,
                 title,
-                url: slug,
+                url: url,
```
Comment (mirhamasala) [inline config/gatsby/plugins/rssFeed.js:48]:
```suggestion
                url,
```

File: src/components/BlogPostPreview.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+import * as styles from "~/src/templates/blog/post.module.scss"
+import * as titleStyles from "~/src/components/blog/post/header.module.css"
+
+const BlogPostPreview = (props) => {
+  const md = props.entry.getIn(["data", "body"])
+  const [html, setHTML] = useState("")
+
+  useEffect(() => {
+    ;(async () => {
```
Comment (mirhamasala) [inline src/components/BlogPostPreview.js:22]: Is this semicolon on purpose?

**Review by mirhamasala — APPROVED** [review]

File: src/components/BlogPostPreview.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+        setHTML(value)
+      } catch (e) {
+        console.error(e)
+      }
+    })()
+  }, [md])
+
+  return (
+    <div className={styles.root} style={{ paddingTop: "10px" }}>
```
[the remark this answers is not in the document]
Comment (gabrielpoca) [reply src/components/BlogPostPreview.js:37]: I'm just trying to override the existing styles and not change any CSS. This is just the preview.

File: src/components/BlogPostPreview.js (hunk 2 above)
Comment (gabrielpoca) [reply src/components/BlogPostPreview.js:22]: Prettier adds it automatically. I think it's to avoid some syntax error because there's a ( after {.

**Pull request merged**